pymeasure/instruments/hp/hp663X2A.py

```diff
@@ -0,0 +1,233 @@
+        """Measure the low voltage level from a sample.""",
+    )
+
+    voltage_max = Instrument.measurement(
+        "MEAS:VOLT:MAX?",
+        """Measure the high voltage level from a sample.""",
+    )
+
+    # Output Trigger related commands
+    def init_single_output_trigger(self):
+        """Set the state of the triggering system to "wait-for-trigger".
+
+        The source will change the output levels to voltage_trigger_level and current_trigger_level.
+        It will return to Idle after a single trigger"""
+        self.write("INIT:NAME TRAN"),
+
+    def enable_cont_output_trigger(self):
+        """Set the state of the triggering system to "wait-for-trigger".
+
+        The source will change the output levels to voltage_trigger_level and current_trigger_level.
+        It will return to Idle after a single trigger"""
+        self.write("INIT:CONT:NAME TRAN, ON"),
+
+    trigger_output = Instrument.setting(
+        "*TRG",
+        """Set the trigger once""",
+    )
+
+    # Display related commands
+    display_enabled = Instrument.control(
+        "DISP?", "DISP %s",
+        """Control the display state.""",
+        validator=strict_discrete_set,
+        values=BOOL_MAPPINGS,
+        map_values=True,
+    )
+
+    display_mode = Instrument.control(
+        "DISP:MODE?", "DISP:MODE \"%s\"",
+        """Control the display, switch between normal and text display modes""",
+        validator=strict_discrete_set,
+        values=["NORM", "TEXT"],
+    )
+
+    displayed_text = Instrument.control(
+        "DISP:TEXT?", "DISP:TEXT \"%s\"",
+        """Control the text displayed on the source's display.
+
+        The text can be up to 12 characters long;
+        any additional characters are truncated my the multimeter.""",
+        get_process=lambda x: x.strip('"'),
+    )
+
+    # System related commands
+    remote_control_enabled = Instrument.control(
+        "SYST: ", "SYST:%s",
+        """Control whether remote control is enabled.""",
+        validator=strict_discrete_set,
+        values={True: "REM", False: "LOC"},
+        map_values=True,
+    )
+
+    error = Instrument.measurement(
+        "SYST:ERR?",
+        """Measure the next error in the buffer (FIFO)""",
+    )
+
+    remote_lock_enabled = Instrument.control(
+        "SYST: ", "SYST:%s",
+        """Control whether the beeper is enabled.""",
+        validator=strict_discrete_set,
+        values={True: "RWL", False: "LOC"},
+        map_values=True,
+    )
+
+    scpi_version = Instrument.measurement(
+        "SYST:VERS?",
+        """Measure the SCPI version of the source.""",
+    )
+
+    self_test_result = Instrument.measurement(
+        "*TST?",
+        """Measure a self-test of the source and return the result.
+
+        Be sure to set an appropriate connection timeout,
+        otherwise the command will fail.""",
+    )
+
+    def write(self, command):
+        """Write a command to the instrument."""
+        super().write(command)
```

Comment:
if you do not define a `write` method, it will automatically use the inherited one (which you call with `super()`), therefore, these lines are superfluous.